src/libs/ReportPreviewActionUtils.ts

```diff
@@ -0,0 +1,105 @@
+import type {OnyxCollection} from 'react-native-onyx';
+import type {ValueOf} from 'type-fest';
+import CONST from '@src/CONST';
+import type {Policy, Report, TransactionViolation} from '@src/types/onyx';
+import {isApprover as isApprovedMember} from './actions/Policy/Member';
+import {getCurrentUserAccountID} from './actions/Report';
+import {arePaymentsEnabled, getCorrectedAutoReportingFrequency, hasAccountingConnections, isAutoSyncEnabled, isPrefferedExporter} from './PolicyUtils';
+import {
+    hasViolations as hasAnyViolations,
+    isClosedReport,
+    isCurrentUserSubmitter,
+    isExpenseReport,
+    isInvoiceReport,
+    isIOUReport,
+    isOpenReport,
+    isPayer,
+    isProcessingReport,
+    isReportApproved,
+    isSettled,
+} from './ReportUtils';
+import {getSession} from './SessionUtils';
+
+function canSubmit(report: Report, violations: OnyxCollection<TransactionViolation[]>, policy?: Policy) {
+    const isExpense = isExpenseReport(report);
+    const isSubmitter = isCurrentUserSubmitter(report.reportID);
+    const isOpen = isOpenReport(report);
+    const isManualSubmitEnabled = getCorrectedAutoReportingFrequency(policy) === CONST.POLICY.AUTO_REPORTING_FREQUENCIES.MANUAL;
+    const hasViolations = hasAnyViolations(report.reportID, violations);
+    return isExpense && isSubmitter && isOpen && isManualSubmitEnabled && !hasViolations;
+}
+
+function canApprove(report: Report, violations: OnyxCollection<TransactionViolation[]>, policy?: Policy) {
+    const isExpense = isExpenseReport(report);
+    const isApprover = isApprovedMember(policy, getCurrentUserAccountID());
+    const isProcessing = isProcessingReport(report);
+    const isApprovalEnabled = policy ? policy.approvalMode && policy.approvalMode !== CONST.POLICY.APPROVAL_MODE.OPTIONAL : false;
+    const hasViolations = hasAnyViolations(report.reportID, violations);
+    return isExpense && isApprover && isProcessing && isApprovalEnabled && !hasViolations;
+}
+
+function canPay(report: Report, violations: OnyxCollection<TransactionViolation[]>, policy?: Policy) {
+    const isReportPayer = isPayer(getSession(), report, false, policy);
+    const isExpense = isExpenseReport(report);
+    const isPaymentsEnabled = arePaymentsEnabled(policy);
+    const isApproved = isReportApproved({report});
```

Comment:
We need to change this condition to:

```
    const isApprovalEnabled = policy ? policy.approvalMode && policy.approvalMode !== CONST.POLICY.APPROVAL_MODE.OPTIONAL : false;
    const isApproved = isReportApproved({report}) || !isApprovalEnabled;
```

So that reports on policies without approvals enabled can be paid